src/icons/folderIcons.ts

```diff
@@ -10,18 +10,17 @@ export const folderIcons: FolderTheme[] = [
         rootFolder: { name: 'folder-root' },
         icons: [
             { name: 'folder-src', folderNames: ['src', 'source', 'sources'] },
-            { name: 'folder-dist', folderNames: ['dist', 'out', 'build', 'release', 'bin'] },
+            { name: 'folder-dist', folderNames: ['dist', 'out', 'build', 'release', 'bin', 'ltximg', 'latex.out'] },
```

Comment:
These two folder names are very specific, not sure if I'll add them to the theme. If people want these icons they can assign them manually in the user settings.

Reply:
I see your point, I forgot this was possible. =)
(just found it in the readme: https://github.com/PKief/vscode-material-icon-theme#folder-associations)